src/toggl_api.h

```diff
@@ -1300,6 +1301,9 @@ extern "C" {
         const char_t *hexColor,
         TogglAdaptiveColor type);
 
+    TOGGL_EXPORT bool toggl_is_timeline_ui_enabled(
```

Comment:
as this is only used on Windows it works, but for cross-platform we should use `bool_t`